contribs/src/main/java/com/netflix/conductor/contribs/storage/config/S3Properties.java

```diff
@@ -25,7 +28,8 @@
     /**
      * The time (in seconds) for which the signed url will be valid
      */
-    private int signedUrlExpirationSeconds = 5;
+    @DurationUnit(ChronoUnit.SECONDS)
```

Comment:
Same as above, `DurationUnit` may not be necessary and the comment needs update.